lib/secure-boot-lib.robot

```diff
@@ -175,113 +144,77 @@ Check Secure Boot In Windows
     ...    Should Contain    ${out}    True
     RETURN    ${sb_status}
 
-Enable Secure Boot
-    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
-    ${device_mgr_menu}=    Enter Submenu From Snapshot And Return Construction
-    ...    ${setup_menu}
-    ...    Device Manager
-    ${secure_boot_menu}=    Enter Submenu From Snapshot And Return Construction
-    ...    ${device_mgr_menu}
-    ...    Secure Boot Configuration
-    Select Attempt Secure Boot Option
-    Save Changes And Reset    2
-
-Disable Secure Boot
-    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
-    ${device_mgr_menu}=    Enter Submenu From Snapshot And Return Construction
-    ...    ${setup_menu}
-    ...    Device Manager
-    ${secure_boot_menu}=    Enter Submenu From Snapshot And Return Construction
-    ...    ${device_mgr_menu}
-    ...    Secure Boot Configuration
-    Clear Attempt Secure Boot Option
-    Save Changes And Reset    2
-
-Enter Custom Secure Boot Options
-    [Documentation]    Sets the Secure Boot Mode to Custom Mode from
-    ...    the Secure Boot Configuration menu and enters
-    ...    the Custom Secure Boot Options menu
-    ...
-    ${out}=    Get Secure Boot Configuration Submenu Construction
-    ${is_standardmode}=    Run Keyword And Return Status
-    ...    Should Contain Match    ${out}    *Standard*
-    IF    ${is_standardmode}
-        Press Key N Times And Enter    2    ${ARROW_DOWN}
-        Read From Terminal Until    Custom Mode
-        Press Key N Times And Enter    1    ${ARROW_DOWN}
-        Read From Terminal
-        Press Key N Times And Enter    1    ${ARROW_DOWN}
+Enter Enroll DB Signature Using File In DB Options
+    [Documentation]    Keyword checks Secure Boot state in Windows.
+    [Arguments]    ${advanced_menu}
+    ${db_opts_menu}=    Enter Submenu From Snapshot And Return Construction
+    ...    ${advanced_menu}
+    ...    DB Options
+    ...    opt_only=${TRUE}
+    ${enroll_sig_menu}=    Enter Submenu From Snapshot And Return Construction
+    ...    ${db_opts_menu}
+    ...    Enroll Signature
+    ...    opt_only=${FALSE}
+    Enter Submenu From Snapshot    ${enroll_sig_menu}    Enroll Signature Using File
+
+Enter Volume In File Explorer
+    [Documentation]    Enter the given volume
+    [Arguments]    ${target_volume}
+    # 1. Read out the whole File Explorer menu
+    ${volumes}=    Get Submenu Construction    opt_only=${TRUE}
+    Log    ${volumes}
+    # 2. See if our label is within these entries
+    ${index}=    Get Index Of Matching Option In Menu    ${volumes}    ${target_volume}    ${TRUE}
```

Comment:
regarding the comment
https://github.com/Dasharo/open-source-firmware-validation/commit/7157a603ab31297546de1aebf71c759554465958#r133696812
Probably if we use keywrod here
`Get Index Of Matching Option In Menu` then you will be able to undo the changes in the keyword itself.

Reply:
@Psotas Please create issue so we can fix that later

Reply:
@macpijan Issue added https://github.com/Dasharo/open-source-firmware-validation/issues/135